content/general/concepts/http/http.md

```diff
@@ -0,0 +1,107 @@
+---
+Title: 'HTTP'
+Description: 'HTTP is used for fetching HTML documents and other Web-based resources.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+Tags:
+  - 'Browsers'
+  - 'Development'
+  - 'Errors'
+  - 'HTTP'
+CatalogContent:
+  - 'learn-intermediate-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Hypertext Transfer Protocol (HTTP) is used for fetching [HTML](https://www.codecademy.com/resources/docs/html) documents and other Web-based resources. It follows a model where a client (most commonly a browser) requests content stored somewhere on a server.
+
+## HTTP/HTTPS
+
+Instead of HTTP, a more secure version, HTTPS, should be used in order to encrypt information sent between the client and server. This encryption is done with Transport Layer Security (or TLS, formerly [SSL](https://www.codecademy.com/resources/docs/general/ssl)). Some benifits of HTTPS include:
```

Comment:
```suggestion
Instead of HTTP, a more secure version, HTTPS, should be used in order to encrypt information sent between the client and server. This encryption is done with Transport Layer Security (or TLS, formerly [SSL](https://www.codecademy.com/resources/docs/general/ssl)). Some benefits of HTTPS include:
```